### ISSUE TITLE
Captcha 

### ISSUE THREAD

**Issue by pierre-not-dev** [alessio-ds/bsc-whales-monitor#4]
Hello ! 

This part of code seems to be striked by Cloudflare / Captcha :

`tn = (requests.get("https://bscscan.com/address/" + token)).text`
`ptn = tn.find("View Token Tracker Page")`
`print(tn)`

Do you know how to avoid it?

**Comment by geeks121** [comment]
did you solve?

**Comment by pierre-not-dev** [comment]
Yes, for this part. But I had to use selenium instead of requests to avoid captcha.

**Comment by geeks121** [comment]
nice.. can you share it? or already there in your fork?

**Comment by geeks121** [comment]
i test add with user agent headers and it works..
here is the code.. 

```
import requests

url = 'https://bscscan.com'
headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36'}

response = requests.get(url, headers=headers)
print(response.content)
```

i tried add headers .. and it solved .. let me try again and if succes i'll try to pull request

**Comment by geeks121** [comment]
![Screenshot_2022-01-07-23-17-44-967_com.termux.jpg](https://user-images.githubusercontent.com/16743443/148573256-053865dc-1043-4c0f-ab3e-6f98a53801df.jpg)

**Comment by geeks121** [comment]
work

**Comment by geeks121** [comment]
btw do you work for other token? the bot only work for cake

**Comment by alessio-ds** [comment]
> i test add with user agent headers and it works.. here is the code..
> 
> ```
> import requests
> 
> url = 'https://bscscan.com'
> headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36'}
> 
> response = requests.get(url, headers=headers)
> print(response.content)
> ```
> 
> i tried add headers .. and it solved .. let me try again and if succes i'll try to pull request

good way to solve! please fork my project, update the code and i'll merge it.



> btw do you work for other token? the bot only work for cake

yes, it does work with every token, even for shitcoins

**Comment by geeks121** [comment]
yes it work after i change the cake from the find in the code .. and it solved now and already merged
